src/data/CassandraBackend.hpp

```diff
@@ -905,6 +944,16 @@ class BasicCassandraBackend : public BackendInterface {
         executor_.write(std::move(statements));
     }
 
+    void
+    writeMPTHolders(std::vector<MPTHolderData> const& data) override
```

Comment:
An edge case, I don't know if it is possible.  If this MPT have 2 payments in one ledger, it will generate 2 MPTHolderData, right?  The executor_.write will call batch write operation which does not guarantee the execution order.  We need to make sure the final one is written.